javascript/src/URDFClasses.js

```diff
@@ -147,9 +147,14 @@ class URDFJoint extends URDFBase {
     }
 
     /* Public Functions */
+    /**
+     * @param {...number|null} values The joint value components to set, optionally null for no-op
+     * @returns {boolean} Whether the invocation of this function resulted in an actual change to the joint value
+     */
```

Comment:
Can we update the README, instead?

Reply:
can we do both? it's nice to have the JSDoc type hints when using an editor that has a JS language server.